clang/lib/Sema/SemaOverload.cpp

```diff
@@ -6226,6 +6226,15 @@ ExprResult Sema::CheckConvertedConstantExpression(Expr *From, QualType T,
   return R;
 }
 
+ExprResult Sema::EvaluateConvertedConstantExpression(Expr *E, QualType T,
+                                                     APValue &Value,
+                                                     Sema::CCEKind CCE,
+                                                     bool RequireInt) {
+
+  APValue PreNarrowingValue;
```

Comment:
It feels a bit gratuitous to create a wrapper just to avoid a temporary variable. Maybe if we used it several times but I just see one use.

Reply:
That's because I wanted to use the same pattern as 
CheckConvertedConstantExpression/BuildConvertedConstantExpression which both defer to a static function.
They probably could all be changed to be members of Sema if we wanted to.
I'd rather not do that in this patch though

Reply:
After offline, i changed it. Let me know if that makes you happier.